coralproject/talk
# [CORL-163] Add feature button to moderation cards

## What does this PR do?

Adds functionality for featuring and un-featuring comments from the moderation queue.
There is a new button to toggle featured state on a comment.

## How do I test this PR?

- Comment on a story
- Head to your moderation queue
- Notice that the story has a new button in top right labelled "Feature"
- Click it to approve/feature the comment
- Check your history and head to single moderation card view
- You can un-feature the comment if you would like
- Feature button should toggle accordingly with appropriate text and styles (i.e. switch to "Feature" vs "Featured" and colouring should change)
